app/views/lit/localizations/_form.html.erb

```diff
@@ -19,5 +19,32 @@
 
   <button class="btn btn-primary btn-sm" type="submit"><%= I18n.t('lit.common.update', default: "Update") %></button>
   <button class="btn btn-default btn-sm cancel" type="reset"><%= I18n.t('lit.common.cancel', default: 'Cancel') %></button>
+
+  <% if Lit::CloudTranslation.provider %>
+    <div class="btn-group pull-right">
+      <%= link_to "Translate using #{Lit::CloudTranslation.provider.name.demodulize}", cloud_translation_path(from: I18n.default_locale, localization_id: @localization.id), remote: true, class: 'btn btn-sm btn-default' %>
+      <button class="btn btn-sm dropdown-toggle" data-toggle="dropdown">
+        <span class="caret"></span>
+        <span class="sr-only">Source languages</span>
+      </button>
+      <ul class="dropdown-menu">
+        <li>
+          <a>Translate from:</a>
+        </li>
+        <li role="separator" class="divider"></li>
+        <% ([I18n.default_locale] + (I18n.available_locales - [I18n.default_locale]) + [:auto]).each_with_index do |source_locale, i| %>
```

Comment:
I took me a moment to understand why are you removing and adding `default_locale`. Wouldn't it be better to do this like this:

```ruby
I18n.available_locales.sort_by { |l| l == I18n.default_locale ? 0 : 1 }
```

? Or extract it to some separate method?